server/fishtest/schemas.py

```diff
@@ -90,6 +90,8 @@ def size_is_length(x):
 user_schema = {
     "_id?": ObjectId,
     "username": username,
+    "firstname": union("", str),
```

Comment:
I wonder if this union is redundant, btw I didn't include any regex, only length limit, because from experience no matter how smart you would be about a regex for names, you will either allow everything or miss the most obscure legal name ever.

Reply:
Yes it is redundant since “” is a string.

The union means that the object should either be equal to “” or else be of type string. This is equivalent to being of type string.

Reply:
Well the limit is more to avoid spam rather than an intrinsic property of the names.. but noted for the future.. I guess this schema is possible because of vtjson, thanks for the package!